.eslintrc.js

```diff
@@ -64,6 +64,7 @@ module.exports = {
       env: {
         es6: true,
         browser: true,
+        'jest/globals': true,
```

Comment:
I'm not sure this is required since this would apply to all files instead of just the test files.

Reply:
Take a look over here for `envs` that are just applied to test files:

https://github.com/redwoodjs/redwood/blob/480e2d612d5637b1905628d5860752421c0ea8a8/packages/eslint-config/shared.js#L95-L103

Reply:
@peterp I extended the files array in that object to include '*.setup.*' files and it worked; no extra envs needed